src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig

```diff
@@ -161,7 +161,7 @@
                   <div class="content-block content-block-pb">
                     <div class="content-title">
                       <p>{{ 'lbl.Drafts'|trans|ucfirst }}
-                        <abbr class="fa fa-info-circle" data-toggle="tooltip" title="{{ 'msg.HelpDrafts'|trans }}"></abbr>
+                        <abbr class="fa fa-info-circle" aria-hidden="true" data-toggle="tooltip" title="{{ 'msg.HelpDrafts'|trans }}"></abbr>
```

Comment:
Can you undo this? A better fix is available in https://github.com/forkcms/forkcms/pull/2320